dashboard/app/views/followers/student_user_new.html.haml

```diff
@@ -6,137 +6,19 @@
 - content_for(:head) do
   %meta{name: "robots", content: "noindex"}/
 
-%script{src: webpack_asset_path('js/followers/student_user_new.js')}
-
-- if current_user || !@section
-  %h2= t('join_section.code.title')
-  %p= t('join_section.code.instructions')
-  - display_captcha = current_user && current_user.display_join_section_captcha?
-  = form_tag(student_user_new_path) do
-    = text_field_tag(:section_code, params[:section_code], placeholder: I18n.t('join_section.code.placeholder'))
-    - if display_captcha
-      = recaptcha_tags callback: "onCaptchaCompletion"
-    %br/
-    = submit_tag t('join_section.code.submit'), class: 'btn btn-primary', disabled: display_captcha, id: 'join_new_section'
-    - if inline_alert
-      %span.inline-alert{ style: 'color: #b94a48; margin-left: 12px'}= inline_alert
-
-- else
```

Comment:
This is the case where we'd show a user the old join section signup form, so we no longer need this